sw/host/tests/chip/jtag/src/main.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use lazy_static::lazy_static;
+use std::collections::HashMap;
```

Comment:
These deps look unused, is this just for the draft?

Reply:
Yes, there is a no of unused imports at the moment.